src/main/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestClientInternal.java

```diff
@@ -157,21 +168,34 @@
 
     this.flushService = new FlushService(this, this.channelCache, this.isTestMode);
 
-    if (ENABLE_PERF_MEASUREMENT) {
+    if (this.enableJmxMetrics) {
```

Comment:
we still want memory and threads information for JVM, it's not useful for customer, but it's useful when we debugging memory issues. And with your change, are we still enable to see the results in the log?

Reply:
We wont be able to see results in log. Should we keep memory and thread related info in separate metrics object then?